src/configure/mod.rs

```diff
@@ -334,6 +337,19 @@ async fn configure(ctx: &Context, param: &mut LoginParam) -> Result<()> {
     e2ee::ensure_secret_key_exists(ctx).await?;
     info!(ctx, "key generation completed");
 
+    let ctx2 = ctx.clone();
+    async_std::task::spawn(async move {
```

Comment:
why are you spawning here? this means we dont actually wait for these operations to finish anywhere before reporting progress 940

Reply:
Configure is so nice and fast currently, I didn't want to make it slow again. If you say that there might be problems by doing it that way, I'll move it into a job (maybe I'll do this in any case because fetch_existing_msgs will have to go into a job too and then we can have one job for both)

Reply:
yes, if it is supposed to run independently it should be a job